make/test.mk

```diff
@@ -195,6 +206,19 @@ endif
 	# as the controller starts (which is a use-case for CRT-231)
 	oc apply -f test/e2e/nstemplatetier-basic.yaml -n $(HOST_NS)
 
+.PHONY: deploy-registration
+deploy-registration:
+ifeq ($(REG_REPO_PATH),)
+	$(eval REG_REPO_PATH = /tmp/codeready-toolchain/registration-service)
+endif
+	-oc new-project $(HOST_NS) 1>/dev/null
```

Comment:
I am not sure why are we using `-` here and also to redirect error to `dev/null`,error code is `2` not `1`

Reply:
Discussing here https://github.com/codeready-toolchain/toolchain-e2e/pull/12#discussion_r336439458